app/l10n.py

```diff
@@ -0,0 +1,12 @@
+from flask import request
+
+import os
+
+TRANSLATIONS = ['en'] + os.listdir('/noi/app/translations')
```

Comment:
It's a little tricky... we figure out which template `.po` files to generate based off of the deployment configs, but those are basically irrelevant now.  We should talk about how to do this when you get in today.